### DISCUSSION TITLE
Mobile: Max button is not working in the MP4 content when playing in portrait mode

### DISCUSSION THREAD

**Discussion by Ajoymaity** [Sunbird-Knowlg/Community#117] [Bugs]
**Steps to reproduce:**-
1. Open staging Sunbird mobile app
2. Login with Guest/ Valid credentials
3. Search and Open MP4 content
4. Click on Max button

Observe
Actual result:-
Max button is not working content is not opening in landscape mode

Note:-
Build version:- 5.1.361
Mobile device:- Redmi 8A Dual

**Comment by ashokreddy1208** [comment]
@Ajoymaity We have connected over a call and tried reproducing the issue with multiple mobile app builds and devices but it is not reproducible.  
we will close this issue as we are unable to replicate this issue
